### ISSUE TITLE
Tabbing (tab navigation with keyboard) is not working in Matomo 4.13 (regression)

### ISSUE THREAD

**Issue by mattab** [piwik/piwik#20285]
Issue reported internally:

A user reported that tabbing is not working well on our dashboard. And I reproduced this. 

The current version of Matomo cannot tab navigate. If you click tab, you get as far as the menu-search-box in the upper left and that’s the end. None of the functions on the dashboard are reachable by tab.

For 4.13, including https://demo.matomo.cloud/, I reproduced this on Firefox, Chromium, and Brave.  

Original user emails from today:

    hi, matomo is not digitally accessible. After login I'm trapped in header, and I can't go any further because autofocus takes my focus - try it yourself, loose mouse ane use only keyboard, you wont be able to do anyting.

    Hi,

    My friend is using only assistive technologies and can't use Matomo because of that.
    test is on your demo version.
    It looks like "tab trap" but actually search text input is stealing focus every second and I can't go beyond first item in menu

**Comment by sgiehl** [comment]
This is most likely a regression of the vue migration. Maybe @diosmosis has an idea on that one.

**Comment by diosmosis** [comment]
Looks like the FocusIf updated() hook is called too often. Looking into it.
